pkg/scheduler/framework/plugins/noderesources/balanced_allocation.go

```diff
@@ -82,12 +82,16 @@ func NewBalancedAllocation(_ runtime.Object, h framework.Handle, fts feature.Fea
 
 // todo: use resource weights in the scorer function
 func balancedResourceScorer(requested, allocable resourceToValueMap, includeVolumes bool, requestedVolumes int, allocatableVolumes int) int64 {
+	// This to find a node which has most balanced CPU, memory and volume usage.
 	cpuFraction := fractionOfCapacity(requested[v1.ResourceCPU], allocable[v1.ResourceCPU])
 	memoryFraction := fractionOfCapacity(requested[v1.ResourceMemory], allocable[v1.ResourceMemory])
-	// This to find a node which has most balanced CPU, memory and volume usage.
-	if cpuFraction >= 1 || memoryFraction >= 1 {
-		// if requested >= capacity, the corresponding host should never be preferred.
```

Comment:
for whatever the reason, if `requested>capacity` will only lower the final score for the specific node, this doesn't mean the node is not capable to run the pod.

Reply:
If this was by design, it's inaccurate.
The only case were the fractions are greater than 1 is when we apply the "default requests". In this case, the fraction is an estimation and not the real usage. Why shouldn't we assume 1 instead of directly giving a 0 score?

Reply:
hmm... make sense to me.